common/terms.html

```diff
@@ -136,11 +136,11 @@
     </dd>
     <dt><dfn data-lt="owned element|owned|owned element's|owned elements">Owned Element</dfn></dt>
     <dd>
-      <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a>, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child.</p>
+      <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a> unless the <abbr title="Document Object Model">DOM</abbr> descendant or any of its ancestors is the target of an <pref>aria-owns</pref> relation, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child. 'Owned elements' can have specific <a href="#scope">required context roles</a> in relation to their <a class="termref" href="#dfn-owning-element">owning elements</a>.</p>
```

Comment:
>  or any of its ancestors is the target of an <pref>aria-owns</pref> relation

I don't understand this restriction. Why is it problematic if an "aria-owned" element's parent is already "aria-owned"?

>  or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child

Are elements owned by the owned child owned?